db/schema.rb

```diff
@@ -1087,6 +1087,16 @@
     t.index ["user_id"], name: "index_user_counters_on_user_id", unique: true
   end
 
+  create_table "user_optional_fields", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.string "label"
```

Comment:
Guessing there should be a `null: false` here since we're validating in Rails anyway?

Reply:
Not a database expert, but if you want I can add it here.